corehq/ex-submodules/casexml/apps/stock/utils.py

```diff
@@ -45,19 +45,33 @@ def state_stock_category(state):
 def get_current_ledger_transactions(case_id):
```

Comment:
Yea, makes sense. I think this is the only place that actually needs data which isn't in the `StockState` (it uses `transaction.report.form_id` in the template): https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py#L95